frontend/src/components/controls/RisDateInput.vue

```diff
@@ -4,23 +4,40 @@ import dayjs from "dayjs"
 import customParseFormat from "dayjs/plugin/customParseFormat"
 import { MaskaDetail, vMaska } from "maska"
 import { computed, ref, watch } from "vue"
-
-const props = defineProps<{
-  /** HTML element ID of the form field. */
-  id: string
-
-  /** Value of the form field. */
-  modelValue?: string
-
-  /** Error validation state of the form field. */
-  hasError?: boolean
-
-  /** Visual size of the form field. */
-  size?: "regular" | "medium" | "small"
-
-  /** Enable or disable editing the form field. */
-  isReadOnly?: boolean
-}>()
+import IconErrorOutline from "~icons/ic/baseline-error-outline"
+
+const props = withDefaults(
+  defineProps<{
+    /** HTML element ID of the form field. */
+    id: string
+    /** Value of the form field. */
+    modelValue?: string
+    /** Visual size of the form field. */
+    size?: "regular" | "medium" | "small"
+    /** Enable or disable editing the form field. */
+    isReadOnly?: boolean
+    /** Label of the form field. */
+    label?: string
+    /** Validation error and message to display. */
+    validationError?: ValidationError
+    /** Label position: 'above' or 'left' */
+    labelPosition?: "above" | "left"
+    /** Custom classes for the container */
+    containerClass?: string
+    /** Custom classes for the label */
+    labelClass?: string
```

Comment:
do we actually need this or is this just the same class list for every `RisDateInput` anyway?

Reply:
I have removed both.